.travis.yml

```diff
@@ -12,6 +12,7 @@ services:
 - redis-server
 
 script:
+- npm run build
```

Comment:
Why `npm run build` is needed here?

Reply:
test may have type errors like trying to access private properties, but the code itself is fine, so this guards against build time errors while the tests are for logical errors only.